src/patterns/molecules/forms/container.hbs

```diff
@@ -0,0 +1,70 @@
+---
+name: Button Container
+description: |
+  The `.mzp-c-button-container` class can be used around a group of buttons. It applies a flexbox context and the default   form spacing. Modifier classes can then be added to change the layout.
+order: 1
+---
+<div class="mzp-c-form">
+  <h2 class="mzp-u-title-xs">Default</h2>
```

Comment:
All of these examples along with the explanatory headlines seems more suited to a demo page. Maybe this example should be simplified and then link to a more complicated demo with all of the options.